index.html

```diff
@@ -53,6 +54,302 @@ <h1>I'm Redwan.<br/>
+                                                <div class="link-log"><img src="images/HTML.png" alt="">  </div>
+                                                <div class="lnk-txt"><a  href="">HTML</a></div> 
+                                            </div>
+                                        </li>
+                                        <li class="links" >
+                                            <div class="btn">
+                                                <div class="link-log"><img src="images/CSS.png" alt="">  </div>
+                                                <div class="lnk-txt"><a  href="">CSS</a></div> 
+                                            </div>
+                                        </li>
+                                    </ul>
+    
+                                </li>
+                                <li class="lang"> 
+                                    <div class="blink-name">
+                                        <div class="nameB">FrameWorks</div>
+                                        <div class="blink-arrow"> &gt; </div>
+                                    </div>
+                                </li>
+                                <li><hr> </li>
+                                <li class="lang"> 
+                                    <div class="blink-name">
+                                        <div class="nameB">Skils</div>
+                                        <div class="blink-arrow"> &gt; </div>
+                                    </div>
+                                </li>
+                                <li><hr> </li>
+                            </ul>
+                        </div>
+                      </div>
+                </div>
+    
+            </div>
+           <div class="contact-us">
+                <div class="contants">
+                    <h2>Contact me</h2>
+                    <p>
+                        If you have an application you 
+                        are interested in developing, a 
+                        feature that you need built or a
+                        project that needs coding.  
+                        I’d love to help with it!</p>
+                </div>
+                <form action="https://formspree.io/f/mnqykova" method="post" class="contact-form">
+                    <input
+                    type="text"
+                    placeholder="Lorem Ipsum"
+                    class="input-client-info"
+                    name="full-name"
+                    minlength="2"
+                    maxlength="30"
+                    required />
+                    <input
+                    type="email"
+                    name="email"
+                    placeholder="loremipsum@gmail.com"
+                    class="input-client-info"
+                    required />
+                    <textarea name="message" id="" maxlength="300" placeholder="Write your message here ..." ></textarea>
```

Comment:
- [ ] I will implore you to add required attribute to the textarea too so that it would not be submittable empty.